include/ada/unicode.h

```diff
@@ -201,6 +201,10 @@ ada_really_inline size_t percent_encode_index(const std::string_view input,
  * Return true if the content was ASCII.
  */
 constexpr bool to_lower_ascii(char* input, size_t length) noexcept;
+
+bool is_valid_identifier_part(const char32_t& c) noexcept;
```

Comment:
I recommend adding comments to every function you create to help us review it better.

Reply:
Sure thing!